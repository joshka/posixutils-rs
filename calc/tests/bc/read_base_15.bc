ibase = 15
0
1
ACDC2
123BAD.4DA
quit
