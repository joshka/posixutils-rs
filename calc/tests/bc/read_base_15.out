0
1
549857
873388.327
